tests/unit/worker_test.ts

```diff
@@ -482,8 +515,10 @@ Deno.test("Worker limit children permissions granularly", async function () {
     readAbsent: "prompt",
     runGlobal: "prompt",
     runFoo: "granted",
+    runFooPath: "granted",
     runBar: "granted",
     runBaz: "granted",
+    runUnresolved: "prompt", // unresolved binaries remain as "prompt"
```

Comment:
This shows a breaking change--any command name that can't be resolved when launching the worker will be excluded from having allow permissions.